### PR TITLE
[web][SIMS-#90] Integrate Vuetify 3.0.alpha

### PR DESCRIPTION
**Pull request by fwpushan**
Integrate Vuetify 3.0.alpha with current web app as main UI layout framework.

Observation:
1. application main view is has some added padding in left/right side.
2. The logo is not loading

Vuetify enable
Screenshot 2021-05-14 at 3 26 36 AM

Previous 

Screenshot 2021-05-14 at 3 28 31 AM

### CODE REVIEW

**Review by guru-aot — APPROVED** [review]

File: sources/packages/web/src/App.vue [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
-  <div>
-    <Toast />
-    <router-view />
-  </div>
+  <v-app>
+    <v-main style="position: auto">
+      <router-view></router-view>
+    </v-main>
+  </v-app>
+  <!-- <router-view></router-view> -->
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/App.vue:7]: Please remove the commented line.

File: sources/packages/web/src/App.vue [hunk 2]
```diff
@@ -1,6 +1,28 @@
 <template>
-  <div>
-    <Toast />
-    <router-view />
-  </div>
+  <v-app>
+    <v-main style="position: auto">
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/App.vue:3]: Can you please mention that "style="position: auto" is a temporary fix.

**Review by andrewsignori-aot — APPROVED** [review]
When we talked about the layout I understood that the main layout was not impacted, but the page looks different.

**Review by andrewsignori-aot — COMMENTED** [review]
When we talked about the layout I understood that the main layout was not impacted, but the page looks different.

**Review by ann-aot — APPROVED** [review]

**Comment by andrewsignori-aot** [comment]
Adjusted BC Logo and left/right spaces.

image

**Pull request merged**